### PR TITLE
chore: Use native Array instead of Ember.A()

### PR DESCRIPTION
**Pull request by raido**
Remove dependency on Ember.Array -> A() for keeping track of tokens.

### CODE REVIEW

File: addon/services/page-title-list.js [hunk 1]
```diff
@@ -251,4 +249,18 @@ export default class PageTitleListService extends Service {
...
+   *
+   * IE11 compatible approach due to lack of Array.find support
+   *
+   * @param {String} id
+   * @private
+   */
+  _findTokenById(id) {
+    return this.tokens.filter((token) => {
+      return token.id === id;
+    })[0];
```
Comment (knownasilya) [inline addon/services/page-title-list.js:264]: Should this check length before accessing the first index?

Comment (raido) [reply addon/services/page-title-list.js:264]: Empty array is always returned - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

Comment (raido) [reply addon/services/page-title-list.js:264]: This will end up as `let arr = []; arr[0] => undefined`

File: addon/services/page-title-list.js [hunk 2]
```diff
@@ -141,12 +139,11 @@ export default class PageTitleListService extends Service {
...
 
-    let tokens = A([...this.tokens]);
-    tokens.removeObject(token);
-    set(this, 'tokens', A(tokens));
+    let tokens = [...this.tokens];
+    tokens.splice(tokens.indexOf(token), 1);
+    set(this, 'tokens', tokens);
   }
 
-  @computed('tokens')
```
Comment (raido) [inline addon/services/page-title-list.js:149]: A step towards Octane approach.

File: addon/services/page-title-list.js (hunk 1 above)
Comment (knownasilya) [reply addon/services/page-title-list.js:262]: Ah, yes, forgot that was how index access worked. 👍

**Pull request merged**